src/lib/autocomplete/autocomplete-trigger.ts

```diff
@@ -364,7 +381,6 @@ export class MatAutocompleteTrigger implements ControlValueAccessor, OnDestroy {
    * stream every time the option list changes.
    */
   private _subscribeToClosingActions(): Subscription {
-    const firstStable = first.call(this._zone.onStable.asObservable());
```

Comment:
This observable was previously re-created every time the panel opened. Now it looks like it's only created when the trigger is created the first time, and will complete after the first event emits. Can you add a test that confirms this still works the second time the panel opens?

Reply:
It will work. Observables are not like promises and can complete on each new subscription. There's no need to recreate them every time. But if you feel it's necessary, I'll write that test.

```
// Go to http://reactivex.io/rxjs/ and open up the JavaScript console to try this code snippet out.
const subject = new Rx.Subject();
// Even though, this observable has a first operator it will emit and complete multiple times.
const obs = subject.first();

obs.subscribe((x) => {
  console.log(x);
}, undefined, () => {
  console.log('complete for the first time');
});
subject.next(1);

console.log('subscribing again');
obs.subscribe((x) => {
  console.log(x);
}, undefined, () => {
  console.log('complete for the second time');
});
subject.next(2);
```

Reply:
Fair enough. If you don't mind writing a quick test, I think it would be good to make sure it doesn't break going forward.

Reply:
Okay open and close the panel twice in 'should emit panel close event when clicking away'.